addons/web_editor/static/lib/odoo-editor/src/utils/utils.js

```diff
@@ -2442,3 +2442,16 @@ export function selectionMatchesInnerText(element) {
 export function isMacOS() {
     return window.navigator.userAgent.includes('Mac');
 }
+
+/**
+ * Removes all ZeroWidthSpace characters from a node (if text) or
+ * its descendants.
+ * It does not remove nodes.
```

Comment:
I suggest (break at 80 chars):
Remove zero-width spaces from the provided node and its descendants.